app/javascript/components/track/iteration-summary/TestsStatusSummary.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import { TestRunStatus } from '../../student/Editor'
+
+function Content({ testsStatus }: { testsStatus: TestRunStatus }) {
+  switch (testsStatus) {
+    case TestRunStatus.QUEUED:
+      return (
+        <>
+          <svg role="presentation" className="icon">
+            <use xlinkHref="#loading" />
+          </svg>
+          <div className="status">Queued</div>
+        </>
+      )
+    case TestRunStatus.PASS:
+      return (
+        <>
+          <div className="dot"></div>
+          <div className="status">Passed</div>
+        </>
+      )
+    case TestRunStatus.FAIL:
+      return (
+        <>
+          <div className="dot"></div>
+          <div className="status">Failed</div>
+        </>
+      )
+    case TestRunStatus.OPS_ERROR:
+    case TestRunStatus.ERROR:
+      return (
+        <>
+          <div className="dot"></div>
+          <div className="status">Errored</div>
+        </>
+      )
+    case TestRunStatus.CANCELLED:
+      return (
+        <>
+          <div className="dot"></div>
+          <div className="status">Cancelled</div>
+        </>
+      )
+    default:
+      return null
+  }
+}
+
+export function TestsStatusSummary({
+  testsStatus,
+}: {
+  testsStatus: TestRunStatus
+}) {
+  return (
+    <div className="tests">
```

Comment:
```suggestion
    <div className="tests" 
            role="status"
            aria-label="Test run status" 
            aria-described-by="iteration-test-run-status"
    >
```

Same as analyzer, but now we need to help the screen reader. `Cancelled` by itself doesn't mean anything.

haven't tested this, but will once live.

Reply:
This does mean that if you want to re-use the component, it should have an id passed in, or the `id` on the page won't be unique. Not important for now.

Reply:
@kntsoriano As there are multiple on a page, you'll need to generate the id by including the iteration_id at the end.

Reply:
(and we might want to look into what happens when we just push the description into the `label`, but for that we need to go through the major ~4 screen-readers and I don't have the capacity to do that at the moment. If that works, we can come back to this and remove the ids/aria-described-by, and instead add `aria-hidden` to the labels)